packages/auth/test/auth.test.ts

```diff
@@ -0,0 +1,54 @@
+import { checkSignInStatus } from "../src/index";
+// import * as fetchMock from "fetch-mock";
+
+const TOMORROW = new Date().setDate(new Date().getDate() + 1);
+
+const oldauth = {
+  email: "casey",
+  token: "XYOl7P..",
+  culture: "en",
+  region: "WO",
+  expires: TOMORROW,
+  allSSL: false,
+  accountId: "uCXeTVfooP4IIcx",
+  role: "account_admin",
+  urlKey: "subdomain",
+  customBaseUrl: "maps.arcgis.com"
+};
+
+/*
+  right now jasmine isn't running these tests in Node.js.
```

Comment:
So, this comment made me think that `checkSignInStatus()` should if no `window` or `document`. 

Given that, it seems like what we want is to run the tests below only in a browser, and then add another test that runs only in node that verifies the above error. Is that possible w/ our test set up?